src/pages/content-edit.js

```diff
@@ -70,6 +70,17 @@ module.exports = {
                         class   : css.publishing,
                         enabled : ctrl.form && ctrl.form.checkValidity()
                     }),
+                    m("div", { class : css.actions }, [
+                        m("a", {
+                                title  : "Preview",
+                                href   : ctrl.schema.preview + m.route.param("id"),
```

Comment:
Since we're now referencing the `id` param in multiple places would you have the controller grab the id and store it as `ctrl.id`?

Nitpicky, I know!